sensor/common/networkflow/service/service_impl.go

```diff
@@ -88,16 +89,23 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 	return s.authFuncOverride(ctx, fullMethodName)
 }
 
+// Leaving this commented out for now. Will not merge like this
```

Comment:
what does "Will not merge like this" mean in this context? Is this removed before merging?

Reply:
The code will either be uncommented or removed, but not left as it is. Given the changes I have made I think it is best to remove it.